core-bundle/src/Filesystem/FilesystemUtil.php

```diff
@@ -103,7 +103,7 @@ private static function doListContentsFromUuids(VirtualFilesystemInterface $stor
             try {
                 $uuidObject = Uuid::isValid($uuid) ? Uuid::fromString($uuid) : Uuid::fromBinary($uuid);
 
-                if (!$item = $storage->get($uuidObject)) {
+                if (!$item = $storage->get($uuidObject, VirtualFilesystemInterface::BYPASS_DBAFS)) {
```

Comment:
@m-vo What do you think about this?

Reply:
The original issue in #7544 was due to a deleted file. If deleted via the back end UI or VFS API, the DBAFS should always be in sync afterwards (as soon as every FS access happens through the VFS, the DBAFS is *always* in sync). I would argue, that any other form of deletion without triggering a sync is a discouraged behavior, that we should not support by circumventing the DBAFS for any other valid access.

Instead of using the flag, I would rather add some error handling that gracefully handles the failing access instead of crashing.

Reply:
Just to add what we discussed in the call: the file might only be missing temporarily. e.g. when you copy the database from a live or staging environment to your local environment, without copying all the files, as you might not care about those (and might be too many anyway).

Or if a file got accidentally deleted or moved via (S)FTP in the live environment This should not cause an error in the front end and it should also not cause the file to be removed automatically from the DBAFS as you would then lose all the references, even when you later restore that file.